.github/workflows/console-api-release.yml

```diff
@@ -25,6 +25,8 @@ jobs:
       packages: write
     with:
       tag: ${{ needs.release.outputs.git_tag }}
+    secrets:
```

Comment:
The workflow should inherit secrets as configured above?

Reply:
ah, missed that sorry